.github/dependabot.yaml

```diff
@@ -20,6 +20,13 @@ updates:
         - "github.com*"
       exclude-patterns:
         - "github.com/golang*"
+    rest-dependencies:
+      patterns:
+        - "*"
```

Comment:
I would just put these under the github-dependencies

Reply:
We already have a group named github-dependencies. Do we want to combine these two?

```
github-dependencies:
      patterns:
        - "github.com*"
      exclude-patterns:
        - "github.com/golang*"
     ```

Reply:
Updated based on my guess :)